mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -37,6 +40,14 @@ func init() {
 		Platforms:   []string{"qemu"},
 		Tags:        []string{"ignition"},
 	})
+	register.RegisterTest(&register.Test{
+		Name:        "coreos.ignition.uniquebootfs",
+		Description: "Verify there is only one bootfs.",
+		Run:         runBootfsFailure,
+		ClusterSize: 0,
+		Platforms:   []string{"qemu"},
+		Tags:        []string{"ignition"},
```

Comment:
We can drop this.

Reply:
Aha! Good to know - somehow I thought we couldn't omit those

Reply:
The tag still looks like it's there.

Reply:
I keep wondering, what `tags` impact?